### DISCUSSION TITLE
Friction and Restitution on CollisionObject2D Nodes

### DISCUSSION THREAD

**Discussion by tautologias345** [godotengine/godot-proposals#6026] [2D]
Drawing on **Transformice**, the higher the body's **friction**, the slower the **RigidBody2D Nodes on Rigid Mode** colliding with this body will move. The higher the body's **restitution**, the higher this body will throw bodies colliding with it. I do the same suggestion for 3D.

**Comment by Hiiamwilliam** [comment]
These are already provided by [PhysicsMaterial](https://docs.godotengine.org/en/3.5/classes/class_physicsmaterial.html).
... provided for `RigidBody2D`,  not `CollisionObject2D`. Same for 3D. 
Also, not sure why the Godot 4 docs state that the `PhysicsMaterial` is used to modify collision properties of `PhysicsBody3D`, it should work both for 2D and 3D.
